sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -1474,6 +1474,31 @@ case class Pivot(
   override protected def withNewChildInternal(newChild: LogicalPlan): Pivot = copy(child = newChild)
 }
 
+/**
+ * A constructor for creating a transpose, which will later be converted
```

Comment:
Can we be more specific on "later" here? like during analysis stage by which rule.

Reply:
Sounds good added